includes/class-article-settings.php

```diff
@@ -103,6 +134,32 @@ public function render_metabox( $post, $args ) {
 		}
 	}
 
+	/**
+	 * Render a custom metabox to check/uncheck whether or not the sharing widget should be hidden
+	 *
+	 * @since 1.0.2
+	 * @param obj $post Post object.
+	 * @param obj $args Arguments object.
+	 */
+	public function render_hide_widget_metabox( $post, $args ){
+
+		$hide_republication_widget = get_post_meta( $post->ID, 'republication-tracker-tool-hide-widget', true );
+
+		$checked = '';
+
+		if( true == $hide_republication_widget ) {
+
+			$checked = 'checked';
+
+		}
+
+		echo '<label>';
+			echo '<input type="checkbox" name="republication-tracker-tool-hide-widget" id="republication-tracker-tool-hide-widget" '.$checked.'>';
+			echo __('Hide the Republication sharing widget on this post?', 'republication-tracker-tool');
+		echo '</label>';
+
```

Comment:
Should this output a warning if the value from the post meta does not match the value from the result of the filter for this post?

#51 describes:

> a comparison between the metadata value and the filter's value for the post that remarks whether the widget is programmatically excluded for the post.

Reply:
@benlk Instead of a warning, I updated the function so if the filter is currently being applied on the post you're editing then you'll see a message stating that instead of the option to check/uncheck it in https://github.com/INN/republication-tracker-tool/pull/54/commits/1002f263b842a31385d6723e6b7f9bb52182f7df.

Reply:
👍